### PR TITLE
ic - Add function in jobs controller for milking the cows

### LINKED ISSUE #12: Add a second "job" similar to the test job that will "Milk the cows".
The "jobs" controller has code that allows the user to start a "test job".

You are going to clone that code to add a job for "milking the cows".

So this code:

Image

You'll copy/paste that, but:

change the name from "Launch Test Job" to "Launch Job to Milk the Cows"
remove the ApiParams (we don't need any params for this job)
Inside, the code will just be as shown below:

Image

ALSO NOTE: the "code to milk the cows" is something that this team might implement, or that might be done by team 3; we'll provide more information about that in the next 2-3 days.

This is backend only. There will be a separate issue for the front end changes.

### PR DESCRIPTION
**Pull request by irenecho75**
In this PR, we add a job for "milking the cows" to JobsController.java. Closes #12

### CODE REVIEW

**Review by akdas16 — DISMISSED** [review]
LGTM

**Review by xxzjh — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

File: src/main/java/edu/ucsb/cs156/happiercows/controllers/JobsController.java [hunk 1]
```diff
@@ -64,4 +64,17 @@ public Job launchTestJob(
           });
     }
 
+    @ApiOperation(value = "Launch Job to Milk the Cows")
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @GetMapping("/launch/milkjob")
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/happiercows/controllers/JobsController.java:69]: This should be a @PostMapping not a @GetMapping

The reason is that GET requests are not supposed to update anything.  But this job will definitely update state on the server.  Therefore, it is supposed to be a POST.

**Review by pconrad — CHANGES_REQUESTED** [review]
Please change GET to POST for the milkjob

File: src/main/java/edu/ucsb/cs156/happiercows/controllers/JobsController.java (hunk 1 above)
Comment (irenecho75) [reply src/main/java/edu/ucsb/cs156/happiercows/controllers/JobsController.java]: Fixed!

**Comment by vivianross06** [comment]
Please link this PR to an issue so that we can see what it closes out. You can do this in a comment by typing "Closes #" and selecting the relevant issue.

**Comment by irenecho75** [comment]
Closes #12

**Comment by vivianross06** [comment]
Looks like typing the comment regarding closing an issue only works when you put it in the initial description of a PR, not a comment. Please do that in the future. If you forget, you can add an issue by clicking on "development" in the menu on the right side of the page, and linking the correct issue there. Please do that for this PR!

**Review by vivianross06 — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM!   Can you deploy this to qa so that I can test it before merging?

**Pull request merged**